src/to-be-in-the-document.js

```diff
@@ -0,0 +1,24 @@
+import {matcherHint, printReceived} from 'jest-matcher-utils'
+import {checkHtmlElement} from './utils'
+
+export function toBeInTheDocument(element) {
+  checkHtmlElement(element, toBeInTheDocument, this)
+
+  return {
+    pass: document.documentElement.contains(element),
```

Comment:
Should we check whether `document` and `document.documentElement` exist? I know `jsdom` will have them, but if I accidentally used this matcher in the `node` environment, it would be nice to get a descriptive error message rather than "cannot read property 'documentElement' of undefined".

Reply:
Hrm, sounds good, agreed on developer experience. It would be nice to get a clear error. Does the following sound good @jgoz?

`document is undefined on global but is required for the toBeInTheDocument matcher`.

`documentElement is undefined on document but is required for the toBeInTheDocument matcher.`

Reply:
👍  the first message should be good enough for both cases.